server/datastore/mysql/migrations/tables/20240709132642_StoreCalendarEventsUUIDAsBinary.go

```diff
@@ -0,0 +1,53 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240709132642, Down_20240709132642)
+}
+
+func Up_20240709132642(tx *sql.Tx) error {
+	// Implementation based on: https://dev.mysql.com/blog-archive/storing-uuid-values-in-mysql-tables/
+
+	if _, err := tx.Exec(`ALTER TABLE calendar_events ADD COLUMN uuid_bin BINARY(16) NOT NULL`); err != nil {
```

Comment:
Curious as to why this new `NOT NULL` doesn't require a `DEFAULT` value.

Reply:
MySQL has an implicit default: https://dev.mysql.com/doc/refman/5.7/en/data-type-defaults.html#data-type-defaults-implicit

Other databases don't.